src/common.rs

```diff
@@ -63,18 +63,24 @@ pub struct LineEntry {
 
 impl LineEntry {
     pub fn is_empty_or_comment(&self) -> bool {
-        let trimmed_string = self.raw_string.trim();
+        self.is_empty() || self.is_comment()
+    }
+
+    pub fn is_empty(&self) -> bool {
+        self.trimmed_string().is_empty()
+    }
 
-        trimmed_string.is_empty() || trimmed_string.starts_with('#')
+    pub fn is_comment(&self) -> bool {
+        self.trimmed_string().starts_with('#')
     }
 
     pub fn get_key(&self) -> Option<String> {
         if self.is_empty_or_comment() {
             return None;
         }
 
-        match self.raw_string.find('=') {
-            Some(index) => Some(self.raw_string[..index].to_owned()),
+        match self.trimmed_string().find('=') {
+            Some(index) => Some(self.trimmed_string()[..index].to_owned()),
```

Comment:
Why are these changes needed?

Reply:
Because if we have this line (it is started with whitespaces) in .env file
```
    ABC=def
```
the key is
```
ABC
```
not
```
    ABC
```

Am I right? If not, this change is wrong.

Reply:
I guess you are right 👍